src/pss/signing_key.rs

```diff
@@ -81,7 +81,7 @@ where
     }
 
     /// Return specified salt length for this key
-    pub fn salt_len(&self) -> usize {
+    pub const fn salt_len(&self) -> usize {
```

Comment:
Does it really make sense for this to be `const fn` on a type with no const constructor?

Reply:
no it does not. I sent this way too late last night. My bad.